features/suzie_searches_by_free_text.feature

```diff
@@ -16,11 +16,25 @@ Feature: Suzy Singer searches by free text
     And a hidden petition exists with action: "The Wombles are profane"
     And an open petition exists with action: "Wombles", closed_at: "10 days from now"
 
+    # waiting for govts response
+    And a petition "Force supermarkets to give unsold food to charities" exists and passed the threshold for a response 1 day ago
+    And a petition "Make every monday bank holiday" exists and passed the threshold for a response 10 days ago
+
+    # having a govt response
+    Given a petition "Spend more money on defence" exists and has received a government response 10 days ago
+    Given a petition "Save the city foxes" exists and has received a government response 1 days ago
+
+    # debated
+    Given a petition "Ban Badger Baiting" has been debated 2 days ago
+    Given a petition "Leave EU" has been debated 18 days ago
+
   Scenario: Search for open petitions
-    When I search for "Wombles"
-    Then I should be on the search results page
-    And I should see "Search results - Petitions" in the browser page title
-    And I should see /For "Wombles"/
+    When I go to the petitions page
+    And I follow "Open petitions"
+    And I fill in "Wombles" as my search term
+    And I press "Search"
```

Comment:
Nitpick: Could we use the `I search for "([^"]*)"? with "([^"]*)"` step to collapse these three steps?